llvm/lib/Target/PowerPC/PPCISelLowering.cpp

```diff
@@ -3365,6 +3365,7 @@ SDValue PPCTargetLowering::LowerGlobalTLSAddressAIX(SDValue Op,
   EVT PtrVT = getPointerTy(DAG.getDataLayout());
   bool Is64Bit = Subtarget.isPPC64();
   bool HasAIXSmallLocalExecTLS = Subtarget.hasAIXSmallLocalExecTLS();
+  bool HasAIXSmallTLSGlobalAttr = false;
```

Comment:
since the two variables `HasAIXSmallLocalExecTLS` and `HasAIXSmallTLSGlobalAttr` are used in `if (IsTLSLocalExecModel || Model == TLSModel::InitialExec) {`

please put 
```
   bool HasAIXSmallLocalExecTLS = Subtarget.hasAIXSmallLocalExecTLS();
   bool HasAIXSmallTLSGlobalAttr = false;
```
   
in 

 `if (IsTLSLocalExecModel || Model == TLSModel::InitialExec) {`